generic/samples/location-header/build.gradle

```diff
@@ -0,0 +1,79 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+apply plugin: 'kotlin'
```

Comment:
Why don't you use [plugins DSL](https://docs.gradle.org/current/userguide/plugins.html)?

Reply:
All samples will need to be updated to this, but not in this PR.